.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: push
+
+jobs:
+  test:
+    name: Go ${{ matrix.go-version }} on Ubuntu
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.12.x, 1.13.x]
+        platform: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Setup Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - uses: actions/cache@v1
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    # See https://github.com/actions/setup-go/issues/14
+    - name: Setup env
+      run: |
+        echo "::set-env name=GOPATH::$(go env GOPATH)"
+        echo "::add-path::$(go env GOPATH)/bin"
```

Comment:
Actually this is for anything that uses binaries from `$GOPATH/bin`, since GitHub Golang action doesn't add that to PATH. In this case it's for invoking golangci-lint, yup.